services/agents/mongodb_test.go

```diff
@@ -54,4 +54,14 @@ func TestMongodbExporterConfig(t *testing.T) {
 	assert.Equal(t, expected.Args, actual.Args)
 	assert.Equal(t, expected.Env, actual.Env)
 	assert.Equal(t, expected, actual)
+
+	t.Run("CheckEmptyPassword", func(t *testing.T) {
```

Comment:
But password is not empty!

Reply:
@AlekSi Oh. This is a typo. There is no reason to check an empty password... But there is a reason to check empty credentials. So if a username is empty, credentials string must be empty too.

Reply:
Yeah, I get it. Just pointed out wrong test name.